http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -1324,18 +1342,13 @@ class GKEInterpreter[F[_]](
       raw"""galaxy.configs.galaxy\.yml.galaxy.admin_users=${userEmail.value}""",
       raw"""galaxy.terra.launch.workspace=${workspaceName}""",
       raw"""galaxy.terra.launch.namespace=${workspaceNamespace}""",
-      // Note most of the below file_sources configs are specified in galaxykubeman,
-      // but helm can't update 1 item in a list if the value is an object.
-      // See https://github.com/helm/helm/issues/7569
-      raw"""galaxy.configs.file_sources_conf\.yml[0].api_url=${config.galaxyAppConfig.orchUrl.value}""",
-      raw"""galaxy.configs.file_sources_conf\.yml[0].drs_url=${config.galaxyAppConfig.drsUrl.value}""",
-      raw"""galaxy.configs.file_sources_conf\.yml[0].doc=${workspaceName}""",
```

Comment:
Assuming most of these params aren't needed anymore..

Reply:
@almahmoud removed these, so I'm assuming that is the case